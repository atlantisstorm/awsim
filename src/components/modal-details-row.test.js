import React from 'react';
import { mount } from 'enzyme';
import ModalDetailsRow from './modal-details-row';

describe('App component', () => {
  it('starts with a count of 0', () => {
    const title = "fishbone";
    const data = "redhot";

    const component = mount(
      <table>
          <tbody>
            <ModalDetailsRow title={title} data={data} />
          </tbody>
      </table>
    );
    expect(component).toMatchSnapshot();
    const titleText = component.find('td:first-child').text();
    expect(titleText).toEqual(title);
    const dataText = component.find('td:last-child').text();
    expect(dataText).toEqual(data);
  });
});