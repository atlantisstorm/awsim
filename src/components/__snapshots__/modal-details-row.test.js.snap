// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component starts with a count of 0 1`] = `
<table>
  <tbody>
    <ModalDetailsRow
      data="redhot"
      title="fishbone"
    >
      <tr>
        <td
          className="title"
        >
          fishbone
        </td>
        <td
          className="data"
        >
          redhot
        </td>
      </tr>
    </ModalDetailsRow>
  </tbody>
</table>
`;
